src/components/Home.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import apiService from "../services/apiService";

```

Comment:
Effectively separating concerns by using a service layer (apiService) to handle API calls is a good approach, this makes the code cleaner and also more maintainable and easier to update or expand.